samza-azure/src/main/java/org/apache/samza/system/eventhub/admin/EventHubSystemAdmin.java

```diff
@@ -90,7 +96,8 @@ private String getNextOffset(String currentOffset) {
 
           long timeoutMs = eventHubConfig.getRuntimeInfoWaitTimeMS(systemName);
           EventHubRuntimeInformation ehInfo = runtimeInfo.get(timeoutMs, TimeUnit.MILLISECONDS);
-          LOG.info(String.format("Adding partition ids=%s for stream=%s", Arrays.toString(ehInfo.getPartitionIds()), streamName));
+          LOG.info(String.format("Adding partition ids=%s for stream=%s. EHRuntimetInfo=%s",
+              Arrays.toString(ehInfo.getPartitionIds()), streamName, printEventHubRuntimeInfo(ehInfo)));
```

Comment:
Can you log the partition level metadata that we are returning including the number of messages in the partition (endSequenceNumber-startSequenceNumber)

Reply:
@lhaiesp : Is this a one-time logging operation or a periodic one?

Reply:
Hi, @vjagadish1989  we log this once before we add these partitionId into streamPartitions. After that we won't go into the same execution branch.

Reply:
makes sense, thanks for explaining.